include/quiche.h

```diff
@@ -183,6 +183,11 @@ void quiche_config_set_cc_algorithm(quiche_config *config, enum quiche_cc_algori
 // Configures whether to use HyStart++.
 void quiche_config_enable_hystart(quiche_config *config, bool v);
 
+// Enables support for receiving DATAGRAM frames.
```

Comment:
This needs to be synced with the Rust documentation. Same for many of the other FFI functions below.